src/lake/Lake/Load/Main.lean

```diff
@@ -189,7 +189,18 @@ def Workspace.updateAndMaterialize
           -- Load the package
           let depPkg ← liftM <| loadDepPackage dep leanOpts reconfigure
           if depPkg.name ≠ dep.name then
-            logWarning s!"{pkg.name}: package '{depPkg.name}' was required as '{dep.name}'"
+            if dep.name = .mkSimple "std" then
+              logError s!"what was the 'std' package has been renamed to '{depPkg.name}'; \
+                users should switch packages depending on new versions of std/{depPkg.name} \
```

Comment:
Why the `src/{...}`? I don't understand what the `/` is signifying.

Reply:
This translates to e.g. `std/batteries`. It is just a way of referring to the renamed package in a hopefully non-confusing way.

Reply:
Ah, nope, that was very confusing. :-)

Suggested text:
````
The `std` package has been renamed to `batteries`, and moved to the `leanprover-community` organization.

Downstream packages should replace
```
require std from git "https://github.com/leanprover/std4" @ "main"
```
in their `lakefile.lean` with
```
require batteries from git "https://github.com/leanprover-community/batteries" @ "main"
```
````

(Yes, I realize not everyone will be using this repo URL, or this branch --- such users are experts and can work out how to adapt.)

Reply:
@semorrison Are you just using the ` ``` ` syntax for clarity? These errors are not formatted in any way. As such, maybe an indent would be clearer?

Reply:
Added a version of your suggestion to the PR. Thanks!